<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWindow</class>
 <widget class="QDialog" name="WelcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome to Monopoly !</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="PlayerNameLayout">
       <item>
        <widget class="QLabel" name="PlayerNameLabel">
         <property name="text">
          <string>Enter player name: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PlayerName"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="PlayerColorLayout">
      <property name="title">
       <string>Select player color: </string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="YellowColor">
         <property name="text">
          <string>Yellow</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="GreenColor">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="BlueColor">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RedColor">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="OpponentsLayer">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Enter number of opponents:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="NumbeOfOpponents">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="StartGame">
      <property name="text">
       <string>Start game !</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
